frontend/src/features/projects/erp/forms/EnhancedReferralCompleteForm.tsx

```diff
@@ -53,9 +55,40 @@ const EnhancedReferralCompleteForm = ({
   const isClearanceNotificationSentOnRequired =
     !formikProps.values.clearanceNotificationSentOn &&
     clearanceNotificationSentOnRequired(formikProps.values.status?.code ?? '');
+  const [showNotificationModal, setShowNotificationModal] = useState(false);
 
   return (
     <Container fluid className="EnhancedReferralCompleteForm">
+      {showNotificationModal && (
+        <GenericModal
+          size={ModalSize.LARGE}
+          message={
+            <>
+              <p>Would you like to send a notification of this status transition?</p>
+            </>
+          }
+          okButtonText="Yes"
+          cancelButtonText="No"
+          handleCancel={(e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+            formikProps.setFieldValue('sendNotifications', false);
+            onClickInErp(e);
+            setShowNotificationModal(false);
+          }}
+          handleOk={(e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+            onClickInErp(e);
+            setShowNotificationModal(false);
+          }}
+        />
+      )}
+
+      {formikProps.values.statusCode === ReviewWorkflowStatus.NotInSpl && (
+        <>
+          <h3>Transition back to In ERP</h3>
+          <Button disabled={isReadOnly} onClick={() => setShowNotificationModal(true)}>
```

Comment:
@jdtoombs can you confirm that this modal works correctly if it used multiple times in the same session (ie. not in SPL -> in ERP -> not in SPL -> in ERP)

Reply:
Modal works! Just looking into the 'Approved for ERP' issue right now